vmm/src/lib.rs

```diff
@@ -2436,6 +2441,7 @@ mod tests {
         assert_eq!(vmm.vm_config.cpu_template, Some(CpuFeaturesTemplate::T2));
 
         // 3. Test update vm configuration after boot.
+        // Also set missing optional parameters to their default value.
```

Comment:
After boot the vm_config cannot be updated so this comment doesn't seem accurate.